pkg/virt-handler/node-labeller/model.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ */
+
+package nodelabeller
+
+import (
+	"sync"
+)
+
+type cpuFeatures map[string]bool
+
+type supportedFeatures struct {
+	sync.RWMutex
```

Comment:
This mutex is probably not needed. All goroutines that read `items` are started after the `items` is written to.